server/src/modules/Question/question.controller.ts

```diff
@@ -0,0 +1,38 @@
+import { Controller, Get, Param, Post, Body, Put, Delete } from '@nestjs/common';
+import { QuestionDto } from './dto/question.dto';
+import { QuestionService } from './question.service';
+
+@Controller('question')
```

Comment:
Probably 'questions' will be better (plural)
with current realization /question endpoint will return list of all questions

Reply:
And its ok, if we want to get specific one we should use /question/id: